templates/patterns/button/button.ui_patterns.yml

```diff
@@ -23,30 +23,35 @@ button:
     splash_page:
       label: "Splash page"
       description: "Special buttons used on the site's splash page."
+    ghost:
+      label: "Ghost"
+      description: "Special buttons used on the site's splash page."
   fields:
+    label:
+      type: "text"
+      label: "Label"
+      description: "Label of the button."
+      preview: "Button"
     type_attribute:
       type: "text"
       label: "Type attribute"
       description: "The buttons type attribute"
       preview: "submit"
-    href:
```

Comment:
Just saw that in ECL buttons are only using the HTML element `button`. I've checked the documentation and, indeed, it seems that buttons are not supposed to link anything anymore. 

This needs to be escalated to the INNO team as, clearly, we will still have just links that are supposed to look as buttons but are simply links, like the "call to action" ones.

Reply:
The ECL 2 itself has an example of such call to action usage of a link, [check here](https://v2--europa-component-library.netlify.com/playground/ec/?path=/story/templates-pages--campaign):

![image](https://user-images.githubusercontent.com/153362/59548970-98801100-8f56-11e9-947f-1f1d3e42f087.png)

That is currently a `<button>`, but there is no form or anything around it, so it's a dead-end:

![image](https://user-images.githubusercontent.com/153362/59548978-d3824480-8f56-11e9-985a-65b9023676b6.png)